### ISSUE TITLE
Dockerfile not building properly

### ISSUE THREAD

**Issue by cansav09** [AlexsLemonade/RNA-Seq-Exercises#15]
### Problem: 
I need to add new installations to the Dockerfile for the scRNA-seq training module, however, before I do that, the Dockerfile as it is isn't building right. 

I started attempting to fix it, but I didn't want to do too much fooling around with it since I don't know exactly why each of these things is necessary. 

It appears the problems arise with installation lines that use apt-key. gpg doesn't install but also MariaDB is another thing having trouble installing. 

#### The error message I get:
```bash
Warning: apt-key output should not be parsed (stdout is not a terminal)
Executing: /tmp/apt-key-gpghome.99NZ47fWnv/gpg.1.sh --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8
gpg: cannot open '/dev/tty': No such device or address
The command '/bin/sh -c apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8' returned a non-zero code: 2
```

**Comment by jaclyn-taroni** [comment]
Let's try and get rid of `ensembldb` which I am pretty sure is the reason we need MariaDB. Can you put the exact (3) lines where you had the issue @cansav09 ?

**Comment by cgreene** [comment]
If we could get rid of `ensembldb` that'd be grand. I also recall that being the only thing that depended on MariaDB.

**Comment by cansav09** [comment]
Okay. I will go through and see where ensembldb is being used and whether we can get around that somehow.

**Comment by cansav09** [comment]
So `ensembldb` supposedly installed fine without `Mariadb`, but I will run through the scripts and make sure things are working for real.

**Comment by cgreene** [comment]
oo, nice!

**Comment by cansav09** [comment]
@jaclyn-taroni is skeptical. For good reason.

**Comment by cansav09** [comment]
These problems were addressed by PR #16 . Closing this issue now.
